lib/src/legacy/value/color.ts

```diff
@@ -22,7 +22,7 @@ export class LegacyColor extends LegacyValueBase<SassColor> {
     }
 
     let red: number;
-    if (!green || !blue) {
+    if (green == undefined || blue == undefined) {
```

Comment:
Use triple equal instead of double equal:

```suggestion
    if (green === undefined || blue === undefined) {
```

Reply:
Got it, just pushed a change to explicitly check for both `undefined` and `null`